.github/workflows/unit_test.yaml

```diff
@@ -37,3 +37,39 @@ jobs:
           pytest --cov=. --cov-report xml tests -vv
       - name: Upload Coverage to Codecov
         uses: codecov/codecov-action@v2
+  gpu_unit_tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [linux.8xlarge.nvidia.gpu]
+        python-version: [3.8]
+        cuda-tag: ["cu11"]
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+      - name: Setup conda env
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniconda-version: "latest"
+          activate-environment: test
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
```

Comment:
should we do what torchsnapshot does for gpu testing here? 

https://github.com/pytorch/torchsnapshot/blob/a5e53e21527dda7f4491d4915fe7e6897fdee4ca/.github/workflows/run_tests.yaml#L56-L70

Reply:
Great suggestion! This makes more clean! 

> should we do what torchsnapshot does for gpu testing here?
> 
> https://github.com/pytorch/torchsnapshot/blob/a5e53e21527dda7f4491d4915fe7e6897fdee4ca/.github/workflows/run_tests.yaml#L56-L70

Great suggestion! This makes more clean!